classes/decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
Sorry, this is not part of the requirements, I reread them and didn't find such a condition, in any case, what would be these setters and getters used for? I'd like to understand so I can geet the picture of your suggestion.

Reply:
**[REQUIRED CHANGES]**

- Hey @Swarzstein, You should add the setter and getter method to read and access the nameable as suggested by @tresorsawasawa, Currently, you've not implemented any of the changes. Kindly fix this, and also note that the review process helps in fostering learning. If nothing is clear enough to you, kindly reach out to the reviewer as they'd explain better in detail instead of not implementing any of the aforementioned changes.

Reply:
@zieeco That's why I mentioned to call me if you considered that the mentioned feature needed to be implemented but I never received any call.

Reply:
View on slack